src/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Material+Icons+Outlined">
+    <title>Document</title>
+</head>
+<body>
+    <section>
+        <div class="top">
+            <h2>Today's To Do</h2>
+            <a href="#">
+                <span class="material-icons-outlined load">sync</span>
+            </a>
+        </div>
+        <div class="addListContainer">
+            <input type="text" class="addList" placeholder="Add to your list">
+            <a>
+                <span class="material-icons-outlined add">subdirectory_arrow_left</span>
+            </a>
+        </div>
+        <small id="warning" class="hide"></small>
+        <div class="listContainer">
+            <ul id ="list">
+                <!-- add dynamic list here... -->
+            </ul>
+            <a class="clearAll" href="#">Clear all completed</a>
```

Comment:
- [ ] In order to move a step forward towards the approval of this project, you will need a function for the `Clear all completed button` (This should be a button and not a link) using array.f filter() method. Please make sure to refer back to the requirement of this project. 
- [ ] At the end, make sure that the updated array of items is stored in the local storage, hence you will need an array of items in a list. Unfortunately, you haven't yet created any list. Please be sure to meet the requirement of this project and submit it once more

   Wish you all the best :pray: